src/sections/InvolvementSection.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+
+import { Box, Center, Wrap } from '@chakra-ui/react';
+import { graphql } from 'gatsby';
+import GetInvolvedCard from 'src/components/GetInvolvedCard';
+import SectionHeader from 'src/components/SectionHeader';
+
+interface Props {
+  data: GatsbyTypes.InvolvmentSectionFragment;
+}
+
+const InvolvmentSection = ({ data }: Props) => {
```

Comment:
I think I might prefer `GetInvolvedSection` just to be consistent with the naming of the `GetInvolvedCard`